docs/community.rst

```diff
@@ -144,6 +144,40 @@ Get started!
 7. Submit a pull request through the GitHub website.
 
 
+Testing methodology
+-------------------
+
+The `tests are the specifications <http://blog.mathieu-leplatre.info/your-tests-as-your-specs.html>`_.
+
+* Each test class represents a situation or feature (e.g. ``class AnonymousCreationTest(unittest.TestCase):``)
+* Each test method represents an aspect of the specification (e.g. ``def test_creation_is_allowed_if_enabled_in_settings(self):``)
+* Each test method is independant
+* The assertions should only reflect the specifications of this test method
```

Comment:
I think I know what you mean here, but I think this sentence is kind of confusing. How about `The assertions should only correspond to the aspect of the specification that this test method is exercising`?